cri/v1alpha2/cri_utils.go

```diff
@@ -1258,3 +1265,21 @@ func toCNIPortMappings(criPortMappings []*runtime.PortMapping) []ocicni.PortMapp
 	}
 	return portMappings
 }
+
+// applyContainerConfigByAnnotation updates pouch container config according to annotation.
+func applyContainerConfigByAnnotation(annotations map[string]string, podSandboxID string, config *apitypes.ContainerConfig, hc *apitypes.HostConfig) error {
```

Comment:
both the `podSandboxID` and `config` are useless right now, how about to remove it?

Reply:
we may apply annotations to the `config` or `hc`

Reply:
@fuweid @zhuangqh  Have removed podSandboxID.